src/type/schema.js

```diff
@@ -13,6 +13,7 @@ import {
   isUnionType,
   isInputObjectType,
   isWrappingType,
+  getNamedType,
```

Comment:
@Yogu Looks great now!

One last issue from me: Can you please remove this line?
It cause Travis build to fail:
![image](https://user-images.githubusercontent.com/8336157/34538483-3569028c-f0d5-11e7-9775-4455ed63ac42.png)